src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -101,20 +102,6 @@ MavlinkMissionManager::~MavlinkMissionManager()
 	orb_unadvertise(_offboard_mission_pub);
 }
 
-unsigned
-MavlinkMissionManager::get_size()
```

Comment:
Why did you remove this? Doesn't this lead to buffer saturation when mission transfers are active?